### PR TITLE
update child content and use info alert box

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
update content on radio button disclosures 

## Testing done
verified locally by running [form](http://localhost:3001/education/apply-for-education-benefits/application/5490/). Select child or spouse on the first page and navigate through the form to the benefits eligibility page. 

## Screenshots
**Spouse**
screen shot 2018-09-06 at 15 48 25
screen shot 2018-09-06 at 15 48 20

**Child**
screen shot 2018-09-06 at 15 48 44
screen shot 2018-09-06 at 15 48 37

## Acceptance Criteria (Definition of Done)
when the correct content is displayed on benefits eligibility page upon benefit selection. child / spouse selections on the first page change this content. 

#### Applies to all PRs

- [x] Provide link to originating GitHub issue, or connected to it via ZenHub

### CODE REVIEW

**Comment by rianfowler** [comment]
@peggygannon The [ticket](https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/12878) only explicitly provides new content for the "child" disclosures. I added screenshots for both the child (new content) and spouse (existing content). 

*For the fry chapter 33 disclosure*: The formatting in the ticket suggests that we want to show a bulleted list on the disclosure. I added this to the child disclosure: 
screen shot 2018-09-06 at 15 48 44

The spouse disclosure doesn't use a bulleted list but has similar content. I can update the spouse disclosure or change the child disclosure to match this formatting: 

screen shot 2018-09-06 at 15 48 25

**Comment by peggygannon** [comment]
Hi, @rianfowler Yes, let's update the formatting for the Fry Spouse to the below: 

@rtluu I want to see if we can go back to Shay as clarify if the 8/1/11 date needs to be in the Spouse tooltips? It wasn't clear from her original request if both spouse and child needs updating. 

*Surviving spouse Fry*
I understand that I am choosing to receive Fry Scholarship benefits instead of any DEA benefits for which I am currently eligible. This includes DEA benefits based on:
* The death of the qualifying individual on this application, or 
* The death of any other individuals not listed on this application, or 
* A spouse who has a permanent and total service-connected disability, or 
* Any other criteria as listed in 38 U.S.C. § 3501(a)(1)

**Comment by peggygannon** [comment]
@rianfowler OK, great. thanks. 

@rtluu thanks for following up on this for us!

**Review by cehsu — APPROVED** [review]

File: src/applications/edu-benefits/5490/components/BenefitSelectionWarning.jsx [hunk 1]
```diff
@@ -5,22 +5,36 @@ export default function BenefitSelectionWarning(chapter, relationship) {
...
+          <li>A parent who has a permanent and total service-connected disability, or</li>
+          <li>Any other criteria as listed in 38 U.S.C. § 3501(a)(1)</li>
+        </ul>
+        based on any other criteria as listed in 38 U.S.C. § 3501(a)(1).
+      </div>);
     } else {
       // spouse
-      warningText = 'I understand that I am choosing to receive Fry Scholarship benefits instead of any DEA benefits for which I am currently eligible. This includes DEA benefits based on the death of the qualifying individual on this application, based on the death of any other individuals not listed on this application, based on a spouse who has a permanent and total service-connected disability, or based on any other criteria as listed in 38 U.S.C. § 3501(a)(1).';
+      warningText = (<div>
+        I understand that I am choosing to receive Fry Scholarship benefits instead of any DEA benefits for which I am currently eligible. This includes DEA benefits based on the death of the qualifying individual on this application, based on the death of any other individuals not listed on this application, based on:
```
Comment (erikphansen) [inline src/applications/edu-benefits/5490/components/BenefitSelectionWarning.jsx]: Should these two "I am" instances also switch to "I'm"?

**Review by erikphansen — APPROVED** [review]

File: src/applications/edu-benefits/5490/components/BenefitSelectionWarning.jsx (hunk 1 above)
Comment (rianfowler) [reply src/applications/edu-benefits/5490/components/BenefitSelectionWarning.jsx]: Probably- we were only asked to review the child section content and decided to update the spouse content to have a bulleted list like the child section.

**Pull request merged**